TwilioVerifySDKTests/Plans/CompleteSuite.xctestplan

```diff
@@ -27,6 +27,9 @@
   },
   "testTargets" : [
     {
+      "skippedTests" : [
```

Comment:
`KeychainLAContextTests` should be executed on IntegrationSuite only, that's why it's ignored here.